travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -56,7 +56,6 @@ enable=api-one-multi-together,
     file-not-used,
     incoherent-interpreter-exec-perm,
     invalid-commit,
-    # javascript-lint,
```

Comment:
you should include the javascript-lint now that we are ignoring the 3rd party modules